finat/piola_mapped.py

```diff
@@ -125,20 +127,30 @@ def basis_transformation(self, coordinate_mapping):
         detJ = coordinate_mapping.detJ_at(bary)
 
         dofs = self.entity_dofs()
-        edofs = self._element.entity_dofs()
+        bfs = self._element.entity_dofs()
```

Comment:
Is `bfs` a good label -- the `entity_dofs` identifies degrees of freedom, which I guess are in exact correspondence to basis functions.

Reply:
`edofs` stood for extended dofs, now here it is useful to think that we have more basis functions than degrees of freedom, and that V.T maps basis functions of the extended reference space to physical nodal basis functions.